specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-07-19/quota.json

```diff
@@ -1043,49 +1026,46 @@
     },
     "EmailAction": {
       "description": "Email Action.",
+      "type": "object",
       "properties": {
         "emailAddress": {
           "description": "The email address for the action.",
           "type": "string"
         }
       }
     },
-    "OnFailure": {
-      "description": "The on failure action for auto quota increase.",
-      "$ref": "#/definitions/Actions"
-    },
-    "OnSuccess": {
-      "description": "The on success action for auto quota increase.",
-      "$ref": "#/definitions/Actions"
+    "EmailActions": {
+      "description": "The email actions.",
+      "properties": {
+        "emailAddresses": {
+          "description": "The list of email actions.",
+          "type": "array",
+          "items": {
+            "description": "The email recipients.",
+            "$ref": "#/definitions/EmailAction"
+          }
+        }
+      }
     },
     "Actions": {
       "description": "The actions for auto quota increase.",
       "type": "object",
       "properties": {
         "emailActions": {
           "description": "The email actions for auto quota increase.",
```

Comment:
these are breaking changes to existing versions. how are we handling conversions between existing resources users created ?

Reply:
Breaking Changes:
There are no active users.
We are currently in Private preview, so only internal users. I will coordinate with them, after update.

There are no users for autoQuotaIncrease, so making breaking changes in data model.

Thanks.

Reply:
There are no users for autoQuotaIncrease api, so making breaking changes in data model.